lib/system/generic/zynqmp_a72/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+collect (PROJECT_LIB_HEADERS sys.h)
+
+add_subdirectory(../xlnx_common ${CMAKE_CURRENT_BINARY_DIR}/../xlnx_common)
+# vim: expandtab:ts=2:sw=2:smartindent
```

Comment:
to remove  `vim`  line ( and you can also remove some other instances)

Reply:
updated

Reply:
vim line still present